client/components/Card/Card.js

```diff
@@ -59,44 +59,52 @@ const SVGWrapper = styled.div`
   }
 `;
 
-class Card extends Component {
-  render () {
-    const { 
-      card: {
-        image, 
-        isFlipped,
-        hasMatched,
-      }, 
-    } = this.props;
+export function Card (props = {}) {
+  const { 
+    card: {
+      image, 
+      isFlipped,
+      hasMatched,
+    },
+    onClick,
+  } = props;
 
-    const file = require(`!!raw-loader!../${image}`);
+  const file = image 
+    ? require(`!!raw-loader!../../${image}`)
+    : '';
 
-    const flippedStyle = isFlipped 
-      ? { transform: 'rotateY(-180deg)' } 
+  const flippedStyle = isFlipped 
```

Comment:
This is probably short enough to have on one line

Reply:
I have done as suggested. But noticed the only reason I had it like this was to match the others and keep the file looking tidy and readable.